### ISSUE TITLE
Acceptance of terms of services is not saved

### ISSUE THREAD

**Issue by RachL** [openfoodfoundation/openfoodnetwork#7999]
## Description

If an instance requires shoppers to see terms of service at checkout, the shopper needs to accept at each checkout, instead of just accepting once until the terms are changing.

This was working before see Filipe's note here: https://github.com/openfoodfoundation/openfoodnetwork/pull/7468#issuecomment-858833058

## Steps to Reproduce

1. Go to`/admin/terms_of_service_files` with super admin access
2. Upload or check that a file is correctly there
3. Try to checkout twice
4. The second time the checkbox is still displayed, but it should not

## Animated Gif/Screenshot

You can use spaghetti farm on FR staging: https://staging.coopcircuits.fr/spaghetti-farm/shop

## Workaround

Accept each time

## Severity

bug-s2: a non-critical feature is broken, no workaround

## Your Environment

* Version used: v.3.8.7
* Browser name and version: All
* Operating System and version (desktop or mobile): Both

**Comment by Matt-Yorkley** [comment]
Super weird! I had a play in FR Staging with the Spaghetti hub. The first time I checked out it didn't save the `terms_and_conditions_accepted_at` on the customer record at all (it was `nil`), but the second time around it actually did save it. I followed the same steps both times (added two products to the cart and checked out). So going back the checkout again meant the checkbox was correctly checked after that...
